### PR TITLE
Compendium update + Contributing.md changes

### PR DESCRIPTION
**Pull request by fmordashev**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,11 +33,23 @@ The easiest workflow is to work through Foundry, and follow the following steps:
 2. Fill out the details, modifiers, etc.
 3. Right-clicking the item in the Items sidebar tab and select Export Data.
 4. Save the JSON file into the appropriate compendium subfolder.
+    * Make sure the name of the JSON follows these simple rules
```
Comment (CptTwinkie) [inline CONTRIBUTING.md:36]: I'm not sure the filename is the important part. It's the "Name" value inside the file which should be like the book. All of the json is renamed by cook, as far as I know.

Comment (fmordashev) [reply CONTRIBUTING.md]: Okay, had a different impression on how that works. Reflected the statements from above in another commit

**Comment by fmordashev** [comment]
Added alien archive letters B and C. JMac gave me the jsons and asked to process them

**Comment by TimToxopeus** [comment]
Thanks! :-)

**Comment by CptTwinkie** [comment]
[Missing from equipment compendium](https://trello.com/c/edLiq41i/64-missing-from-equipment-compendium)

**Pull request merged**